README.md

```diff
@@ -111,7 +111,7 @@ You can deploy this project using:
 
 <!-- AUTHORS -->
 
-## 👤singular <a name="authors"></a>
+## 👤Author <a name="author"></a>
 
 > Mention all of the collaborators of this project.
```

Comment:
- Good job, @ahmed025566 👏 
- [ ] [OPTIONAL] If you notice, this template is trying to let you know what exactly should be where. Since this line has no use to users or visitors of your page, I suggest you remove these texts or replace it with a more descriptive and customized messages about what the section is about, which in this case is introducing the author. E.g. you could say, "This is a solo project and you can find my contact details below in case you want to reach out:....."